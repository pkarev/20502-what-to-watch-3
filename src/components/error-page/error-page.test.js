import React from 'react';
import renderer from 'react-test-renderer';
import ErrorPage from './error-page.jsx';

it(`Render ErrorScreen`, () => {
  const tree = renderer
  .create(<ErrorPage/>)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
