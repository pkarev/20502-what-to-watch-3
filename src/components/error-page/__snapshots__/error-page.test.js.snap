// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ErrorScreen 1`] = `
<section>
  <p>
    Упс, ошибочка!!!
  </p>
  <p>
    Попробуйте позднее
  </p>
</section>
`;
